packages/eslint-plugin-pf-codemods/src/rules/v6/cardRemoveVariousProps/card-remove-various-props.md

```diff
@@ -0,0 +1,18 @@
+### card-remove-various-props [(#10056)](https://github.com/patternfly/patternfly-react/pull/10056)
+
+The isSelectableRaised, isDisabledRaised, hasSelectableInput, selectableInputAriaLabel, onSelectableInputChange, isFlat, and isRounded props have been removed
```

Comment:
These are more nits so I'll leave it up to you what you prefer:

- adding "from Card" at the end to match how we've generally wrote these messages (even though the rule name states it and it'll be flagged on the applicable component when codemods are ran)
- an unordered list of the props instead of comma separated, e.g. `The following props have been removed from Card: ...`